tests/submission/test_endpoints.py

```diff
@@ -0,0 +1,922 @@
+import contextlib
+import json
+import os
+
+import boto
+import pytest
+from flask import g
+from gdcdatamodel import models as md
+from gdcdictionary import gdcdictionary
+from moto import mock_s3
+
+from sheepdog.transactions.constants import (
+    CACHE_CASES,
+)
+
+from sheepdog.transactions.upload import UploadTransaction
+from tests.submission.utils import data_fnames, patch_indexclient
+from ..auth_mock import Config as auth_conf
+
+
+definitions = gdcdictionary.resolvers['_definitions.yaml'].source
```

Comment:
Should avoid using `gdcdictionary` since the dependency needs to be removed

Reply:
fixed